src/osversion/VersionTranslator.cpp

```diff
@@ -9694,6 +9698,44 @@ namespace osversion {
         ss << newObject;
         m_refactored.emplace_back(std::move(object), std::move(newObject));
 
+      } else if (iddname == "OS:ZoneVentilation:DesignFlowRate") {
+
+        // 1 Field has been added from 3.9.0 to 3.10.0:
+        // -------------------------------------------
+        // * Density Basis * 26
+        auto iddObject = idd_3_10_0.getObject(iddname);
+        IdfObject newObject(iddObject.get());
+
+        for (size_t i = 0; i < object.numFields(); ++i) {
+          if ((value = object.getString(i))) {
+            newObject.setString(i, value.get());
+          }
+        }
+
+        newObject.setString(26, "Outdoor");
+
+        m_refactored.push_back(RefactoredObjectData(object, newObject));
+        ss << newObject;
+
+      } else if (iddname == "OS:ZoneVentilation:DesignFlowRate") {
```

Comment:
Is this supposed to be "OS:Space:Infiltration:DesignFlowRate"?

Reply:
most definitely, was in a rush yesterday...